packages/@o3r/localization/builders/helpers/localization.generator.ts

```diff
@@ -123,14 +124,18 @@ export class LocalizationExtractor {
       })
       .map(({ref}) => {
         if (!ref.startsWith('.')) {
+          if ((this.options?.libraries || []).some((lib) => ref.startsWith(lib))) {
```

Comment:
it seems `this.options.libraries` cannot be undefined
```suggestion
          if (this.options?.libraries.some((lib) => ref.startsWith(lib))) {
```
?

Reply:
options: Partial<LocalizationExtractorBuilderSchema>
so it can be undefined